certified-connectors/Fliplet/README.md

```diff
@@ -0,0 +1,35 @@
+# Fliplet integration connector
```

Comment:
@squallstar please include all required section in the readme file. please follow our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and for raw view for code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).